tools/site-creator/create-site.js

```diff
@@ -42,23 +42,23 @@ async function checkCodeBus(data) {
 
 async function previewOrPublishPages(data, action, setStatus) {
   const parent = getDestinationPath(data.repo, data.org);
+  const libraryPath = `${parent}/docs/library`;
+  const daInternalPath = `${parent}/.da/`;
 
   const label = action === 'preview' ? 'Previewing' : 'Publishing';
 
   const opts = { method: 'POST', headers: { Authorization: `Bearer ${token}` } };
 
   const callback = async (item) => {
     if (item.path.endsWith('.svg') || item.path.endsWith('.png') || item.path.endsWith('.jpg')) return;
+    // skip publishing library
+    if (item.path.startsWith(libraryPath) || item.path.startsWith(daInternalPath)) return;
     setStatus({ message: `${label}: ${item.path.replace(parent, '').replace('.html', '')}` });
     const aemPath = item.path.replace(parent, `${parent}/main`).replace('.html', '');
     const resp = await fetch(`${AEM_ORIGIN}/${action}${aemPath}`, opts);
     if (!resp.ok) throw new Error(`Could not preview ${aemPath}`);
   };
 
-  // Get the library
-  crawl({
-    path: `${parent}/.da`, callback, concurrent: 5, throttle: 250,
-  });
```

Comment:
What did this do that we can just remove it?

Reply:
It was explicitly publishing `.da`... which is the opposite of what we want. It's a left-over from somewhere I copied this from.